hbp_knowledge/pp2a/seshacharyulu2013.bel

```diff
@@ -203,80 +242,118 @@ a(CHEBI:cantharidin) -| act(p(FPLX:PPP2))
+UNSET Confidence
 
 SET Evidence = "Endogenous CIP2A (cancerous inhibitor of PP2A) inhibits PP2Ac
 activity via interacting with c-Myc (Ser62) and stabilizes it from proteolytic degradation
 [49]. "
+SET Confidence = "Medium"
+p(HGNC:CIP2A) -> composite(p(HGNC:CIP2A),p(HGNC:MYC))
+composite(p(HGNC:CIP2A),p(HGNC:MYC)) -| deg(p(HGNC:MYC))
+p(HGNC:MYC) -| act(p(FPLX:PPP2C))
 p(HGNC:CIP2A) -| act(p(FPLX:PPP2C))
+UNSET Confidence
 
 SET Evidence = "Recently, a major indirect activation of PP2A by inhibiting CIP2A at both the
 transcriptional and translational levels through the drug bortezomib was shown in triple
 negative breast cancer cells [50]."
+SET Confidence = "Medium"
+SET MeSHDisease = "Breast Neoplasms"
 a(CHEBI:bortezomib) -| p(HGNC:CIP2A)
+a(CHEBI:bortezomib) -| r(HGNC:CIP2A)
 a(CHEBI:bortezomib) -> act(p(FPLX:PPP2C))
+UNSET MeSHDisease
+UNSET Confidence
 
 SET Evidence = "Methylation [52] and phosphorylation [53] are two major
 modifications that have been shown to modulate PP2A catalytic efficiency."
+SET Confidence = "High"
 p(FPLX:PPP2, pmod(M)) -- act(p(FPLX:PPP2))
 p(FPLX:PPP2, pmod(Ph)) -- act(p(FPLX:PPP2))
+UNSET Confidence
 
+#can be annotated with respect to shi2009 subunit classification
 SET Evidence = "Phosphorylation of Y307 by
 receptor associated tyrosine kinases effectively decreases the PP2A activity by inhibiting the
 interaction of PP2Ac with the PP2A-PR55/PR61 subunit [54]."
+SET Confidence = "Low"
 p(FPLX:PPP2, pmod(Ph,Tyr,307)) -| act(p(FPLX:PPP2))
+#p(FPLX:PPP2, pmod(Ph,Tyr,307)) -| complex(p(HGNC:PPP2R2A),p(HGNC:PPP2R5B))
+UNSET Confidence
 
+#annotation of the regulatory and core dimers
 SET Evidence = " The addition of a methyl group by LCMT1 at L309 enhances the binding affinity
 of the core dimer (A&C subunit) toward distinct regulatory subunits and provides specific
 activity to the holoenzyme [56]."
+SET Confidence = "Low"
 p(HGNC:LCMT1) -> p(FPLX:PPP2, pmod(M,Leu,309))
 p(FPLX:PPP2, pmod(M,Leu,309)) -> act(p(FPLX:PPP2))
+UNSET Confidence
 
 SET Evidence = " Akt plays an
 important role in cell growth, proliferation and apoptosis."
+SET Confidence = "High"
 p(FPLX:AKT) reg bp(GO:"cell growth")
 p(FPLX:AKT) reg bp(GO:"cell population proliferation")
 p(FPLX:AKT) reg bp(GO:"apoptotic process")
+UNSET Confidence
 
 SET Evidence = "Phosphorylation at Thr-308 and
 Ser-473 leads to activation of Akt and it was found that deregulation of Akt is associated
 with various human malignancies. "
+SET Confidence = "High"
 p(FPLX:AKT, pmod(Ph,Thr,308)) -> act(p(FPLX:AKT))
 p(FPLX:AKT, pmod(Ph,Ser,473)) -> act(p(FPLX:AKT))
+UNSET Confidence
 
+#core enzyme wih b55 together
 SET Evidence = "Co-immunoprecipitation and in-vitro pull down assay
 using pro-lymphoid FL5.12 cells showed a direct association of the PP2A-B55 holoenzyme
 with Akt, which selectively regulates phosphorylation of Akt at Thr-308 and regulates cell
 proliferation and survival [58]."
-p(FPLX:PPP2) -| p(FPLX:AKT, pmod(Ph,Thr,308))
```

Comment:
PP2A-B55 holoenzyme: PP2A holoenzyme involving the regulatory subunit Bα
so it's a complex of different proteins, where one of them is the Bα subunit. Because the exact components of this holoenzyme weren't mentioned i sticked with PPP2, but I think it would be better to encode it as a complex out of PPP2 and its regulatory subunit...

Reply:
But again, there are several genes that can encode this subunit (alpha, beta, gamma, delta isoform)
So I'm not sure about this